internal/driver/commands.go

```diff
@@ -196,16 +196,16 @@ var pprofVariables = variables{
 		"Matching includes the function name, filename or object name.")},
 	"tagfocus": &variable{stringKind, "", "", helpText(
 		"Restrict to samples with tags in range or matched by regexp",
-		"Discard samples that do not include a node with a tag matching the regexp (or within range).",
-		"Can specify both the name and value of the tag which should be matched with notation like -tagignore name=value",
+		"Discard samples that do not include a node with tag matching regexp or range.",
```

Comment:
This line doesn't seem to add a lot of information compared to what the previous one says?

Also lines are still pretty long, can we do a bit shorter?

I think here is the text that I would do:
```
Restricts to samples with tags in range or matched by regexp
Use name=value syntax to limit the matching to a specific tag.
Numeric tag filter examples: 1kb, 1kb:10kb, memory=32mb:
String tag filter examples: foo, foo.*bar, mytag=foo.*bar
```

Let's also make the verb in the first sentence consistently 3rd person, not imperative, where appropriate.